samples/notebooks/dotnet/8-chatGPT-with-DALL-E-2.ipynb

```diff
@@ -0,0 +1,194 @@
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "[An image of a friendly waving hand.]\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "var chat = (OpenAIChatHistory)chatGPT.CreateNewChat(\n",
```

Comment:
suggestion: put this string inside a `systemMessage` variable to use the same terminology as the ChatCompletion API